src/main/java/com/Acrobot/ChestShop/Listeners/Economy/Plugins/VaultListener.java

```diff
@@ -182,7 +179,7 @@ public void onCurrencyHoldCheck(CurrencyHoldEvent event) {
         }
 
         //String lastSeen = NameManager.getLastSeenName(event.getAccount());
-		OfflinePlayer lastSeen = Bukkit.getOfflinePlayer(event.getAccount());
+		    OfflinePlayer lastSeen = Bukkit.getOfflinePlayer(event.getAccount());
```

Comment:
I realise that these lines use tabs instead of spaces but this is worse. I also would prefer it if only code was changed that actually stand in relation to the goal of the commit and that style fixes are separate.

Reply:
I can remove the style fixes as I'm not concerned about them in code I didn't add.